experimental/dds/tree2/src/test/testChange.spec.ts

```diff
@@ -91,3 +102,82 @@ describe("TestChange", () => {
 		assert.deepEqual(normal, codec.decode(codec.encode(normal)));
 	});
 });
+
+interface TestChangeContent {
+	inputContext: number[];
+	value: number;
+}
+
+type TestChangeTestState = FieldStateTree<TestChangeContent, TestChange>;
+
+function rebaseComposed(
+	metadata: RevisionMetadataSource,
+	change: TestChange,
+	...baseChanges: TaggedChange<TestChange>[]
+): TestChange {
+	baseChanges.forEach((base) => deepFreeze(base));
+	deepFreeze(change);
+
+	const composed = TestChange.compose(baseChanges);
+	const rebaseResult = TestChange.rebase(change, composed);
+	assert(rebaseResult !== undefined, "Shouldn't get undefined.");
+	return rebaseResult;
+}
+
+/**
+ * See {@link ChildStateGenerator}
+ */
+const generateChildStates: ChildStateGenerator<TestChangeContent, TestChange> = function* (
+	state: TestChangeTestState,
+	tagFromIntention: (intention: number) => RevisionTag,
+	mintIntention: () => number,
+): Iterable<TestChangeTestState> {
+	const { value, inputContext: context } = state.content;
+	for (let i = value + 1; i < 4; i++) {
+		const intention = mintIntention();
+		const change = TestChange.mint(context, i);
+		yield {
+			content: { value: i, inputContext: change.outputContext },
+			mostRecentEdit: {
+				changeset: tagChange(change, tagFromIntention(intention)),
+				description: `Set${i},${context}`,
+				intention,
+			},
+			parent: state,
+		};
+
+		for (let j = 1; j < value + 1; j++) {
+			const inverseChange = TestChange.mint(context, -i);
+			yield {
+				content: { value: i, inputContext: change.outputContext },
+				mostRecentEdit: {
+					changeset: tagChange(inverseChange, tagFromIntention(intention)),
+					description: `Inverse${j},${context}`,
+					intention,
+				},
+				parent: state,
+			};
+		}
+	}
+};
+
+describe("TestChange - Rebaser Axioms", () => {
+	describe("Exhaustive suite", () => {
+		runExhaustiveComposeRebaseSuite(
```

Comment:
I think it would make sense to add an optional `options` parameter to `runExhaustiveComposeRebaseSuite` so that one can disable specific categories of test.
That would enable us to un-skip the `"Rebase over compose"` suite and let optional field opt out of it. Feel free to tackle that in a separate PR if you prefer.